### ISSUE TITLE
Matek 411 wing and INAV 2.1.0 RC3 Mixer problems

### ISSUE THREAD

**Issue by Kaffiflight** [iNavFlight/inav-configurator#706]
Since three days I tried to make a wing ready with a Matek 411 wing and INAV 2.1.0. After many failures I found out the servomixer is not loading correctly. I can reproduce that issue, and I crosschecked it with 2.0.1, this does not happen there.
As follwos: Flashed, all settings to default (with "defaults" in CLI). Reboot. Go to mixertab, set "Airplane - Flying wing", load mixer. Save and reboot.
Check Mixer: It shows nonsense. Check CLI: It shows several rows of mixersettings, where only 4 lines should be.
As stated above, it works with INAV 2.0.1

**Comment by DzikuVx** [comment]
@Kaffiflight CLI dump, screeshots, exact version of Configurator and firmware please

**Comment by shellixyz** [comment]
@Kaffiflight I guess you are using an older version of the configurator with iNav 2.1.0. You need to use at least version 2.1.4 of the configurator with 2.1.0.

**Comment by Kaffiflight** [comment]
Thank you for your interest and feedback. I can tell I found the reason why.
Yes, I used the configurator 2.1.3, but then just when it came out, the 2.1.4.
The Problem was elsewhere: 
I flashed with a hexfile from my hardisk that I downloaded before (2.1.0 RC3 for the Matek 411, standard file). It flashed without showing any Problems.
Yesterday I downloaded the same file again, overwriting the old one. Now it works like it should be. Obviously there was a faulty hexfile on my PC - must have happened during download. I flashed with it maybe 10 times on 2 different PCs, always with the same result. Not showing that anything was wrong. 
Thanks.
